src/components/MarketMySharesForm/index.js

```diff
@@ -444,7 +444,7 @@ MarketMySharesForm.propTypes = {
   market: marketShape,
   selectedSellAmount: PropTypes.string,
   limitMargin: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  marketShares: PropTypes.arrayOf(PropTypes.object),
```

Comment:
I think we have a marketShares Proptypes, if you want you can change it here to be `PropTypes.arrayOf(marketSharePropType)`

Reply:
Oops, I mean `marketShares: PropTypes.objectOf(marketSharePropType),`.